packages/orchestrator/lib/server.ts

```diff
@@ -23,21 +23,10 @@ export const getServer = (scheduler: Scheduler, eventEmmiter: EventEmitter): Exp
 
     server.use(express.json({ limit: '10mb' }));
 
-    // Logging middleware
-    server.use((req: Request, res: Response, next: NextFunction) => {
-        const originalSend = res.send;
-        res.send = function (body: any) {
-            if (res.statusCode >= 400) {
-                logger.error(`${req.method} ${req.path} ${res.statusCode} -> ${JSON.stringify(body)}`);
-            }
-            originalSend.call(this, body) as any;
-            return this;
-        };
-        next();
-        if (res.statusCode < 400) {
-            logger.info(`${req.method} ${req.path} -> ${res.statusCode}`);
-        }
-    });
+    // Log all requests
+    if (process.env['ENABLE_REQUEST_LOG'] !== 'false') {
+        server.use(requestLoggerMiddleware({ logger }));
+    }
```

Comment:
`requestLoggerMiddleware` doesn't log the body in case of error like the previous code was doing. Are we ok losing this functionality?

Reply:
I haven't used this feature, happy to put it back if you feel it's important.

Reply:
not super important. we can re-add it if needed